lib/diskutil.js

```diff
@@ -151,7 +129,15 @@ const list = (callback) => {
     });
 
     asyncMap(tasks, (devicePath, next) => {
-      run('diskutil', [ 'info', '-plist', devicePath ], next);
+      exec('diskutil', [ 'info', '-plist', devicePath ], (error, stdout) => {
+        let data = null;
+        try {
+          data = plist.parse(stdout);
```

Comment:
What if `exec` yields back an error? We will still attempt to parse `stdout?`

Reply:
I.e. see https://github.com/resin-io-modules/drivelist/pull/255/files#diff-c045d0de9bd65b6a0dd99604dd7b0742R114